src/plugins/navigation/index.ts

```diff
@@ -10,18 +10,33 @@ import backHTML from './templates/back.html?raw';
 export default createPlugin({
   name: () => t('plugins.navigation.name'),
   description: () => t('plugins.navigation.description'),
-  restartNeeded: true,
+  restartNeeded: false,
   config: {
     enabled: true,
   },
   stylesheets: [style],
-  renderer() {
-    const forwardButton = ElementFromHtml(forwardHTML);
-    const backButton = ElementFromHtml(backHTML);
-    const menu = document.querySelector('#right-content');
+  renderer: {
+    start() {
+      const forwardButton = ElementFromHtml(forwardHTML);
+      const backButton = ElementFromHtml(backHTML);
+      this.waitForElem('#right-content').then((menu: HTMLElement) => {
+        menu.prepend(backButton, forwardButton);
+      });
+    },
+    stop() {
+      document.querySelector('[tab-id=FEmusic_back]')?.remove();
+      document.querySelector('[tab-id=FEmusic_next]')?.remove();
+    },
+    waitForElem(selector: string): Promise<Element> {
```

Comment:
`waitForElem` is not needed - the plugin for the renderer is loaded after a DOM is renderered, so `#right-content` is always present.